src/ImageRunModal.jsx

```diff
@@ -687,7 +687,44 @@ export class ImageRunModal extends React.Component {
                 <Tabs activeKey={activeTabKey} onSelect={this.handleTabClick}>
                     <Tab eventKey={0} title={<TabTitleText>{_("Details")}</TabTitleText>} className="pf-c-form pf-m-horizontal">
                         { this.props.userServiceAvailable && this.props.systemServiceAvailable &&
-                        <FormGroup isInline hasNoPaddingTop fieldId='run-image-dialog-owner' label={_("Owner")}>
+                        <FormGroup isInline hasNoPaddingTop fieldId='run-image-dialog-owner' label={_("Owner")}
+                                   labelIcon={
+                                       <Popover aria-label={_("Owner help")}
+                                          enableFlip
+                                          bodyContent={
+                                              <>
+                                                  <TextContent>
+                                                      <Text component={TextVariants.h4}>{_("System")}</Text>
+                                                      <TextList>
+                                                          <TextListItem>
+                                                              {_("Ideal for running services")}
+                                                          </TextListItem>
+                                                          <TextListItem>
+                                                              {_("Resource limits can be set")}
+                                                          </TextListItem>
+                                                          <TextListItem>
+                                                              {_("Checkpoint/restore support")}
+                                                          </TextListItem>
+                                                          <TextListItem>
+                                                              {_("Ports under 1024 can be forwarded")}
+                                                          </TextListItem>
```

Comment:
IIRC there are more network related differences but I couldn't find them? IIRC host networking but we don't offer that.

Reply:
It's fine. We have a lot of the most common differences shown already. :+1: